replay-assets/replay_command_handlers.js

```diff
@@ -474,8 +475,6 @@ function Target_topFrameLocation() {
     }
     throw e;
   }
-
-  return { location: createProtocolLocation(location)[0] };
```

Comment:
This is mildly concerning, I think the "location" reference here is to the global window.location for the initial window so we're not returning correct results here.  I don't know what ramifications that has, if any.